utilities/rocAL/rocAL_unittests/pixel_comparison/image_comparison.py

```diff
@@ -0,0 +1,102 @@
+from PIL import Image
+import cv2 as cv
+import os
+import sys
+import datetime
+import logging
+
+timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S_")
+logging.basicConfig(filename = '../log_file_' + timestamp + '.log', level = logging.INFO)
+if( len(sys.argv) < 3 ):
+    logging.error('Pass atleast 2 arguments (python image_comparison.py <ref_output_folder_path> <rocal_ouput_folder_path>')
```

Comment:
@fiona-gladwin Please add an example to use this script in the README

Reply:
The testAllScript.sh automatically runs the python file should I add the example in README for that?

Reply:
@fiona-gladwin No need then